skdm-lib/SpatialKeyDataManager.cs

```diff
@@ -1256,6 +1256,19 @@ public string ZipFiles(string[] files)
 			{
 				using (var zipFileStream = new FileStream(zippath, FileMode.Create))
 				{
+					if (files.Length == 1 && Path.GetExtension(files[0]).Equals(".zip"))
```

Comment:
Case sensitivity?

Reply:
Updated. converted the extension string to lowercase before checking for equality